.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
This should not be committed in a project.

It would lead to gitignore obesity if everyone adds their IDE things

If you have this need, please take a look at git ignore global: `core.excludesfile`

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.
> https://git-scm.com/docs/gitignore

Reply:
Makes sense ... will expunge that one ..